charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -258,7 +258,7 @@ data:
                   if container.args[0:3] == ["airflow", "tasks", "run"]:
                       container.command = ["tini", "--"]
                       new_args = ["bash", "-c"]
-                      command_str = "/entrypoint " + ' '.join([str(arg) for arg in container.args]) + " " + log_cmd + "; curl -fsSL -XPOST {{ .Values.global.loggingSidecar.terminationEndpoint }}"
+                      command_str = "/entrypoint " + ' '.join([str(arg) for arg in container.args]) + " " + log_cmd + "; curl -fsSL --retry 5 --max-time 10 --retry-connrefused -XPOST {{ .Values.global.loggingSidecar.terminationEndpoint }}"
```

Comment:
@danielhoherd should we make this configurable or let it be defaults

Reply:
@pgvishnuram @danielhoherd 
This works occasionally,not a correct bug fix

Reply:
Okay - let me test this with multiple short lived tasks

Reply:
I really like this `--retry 5 --max-time 10 --retry-connrefused` addition. This seems like a fantastic way to handle this problem.